q2_feature_classifier/_skl.py

```diff
@@ -21,7 +21,8 @@
             'non_negative': True}],
           ['classify',
            {'__type__': 'custom.LowMemoryMultinomialNB',
-            'alpha': 0.01}]]]]
+            'alpha': 0.01,
+            'chunk_size': 20000}]]]]
```

Comment:
Sorry, I forgot to mention that the error that led to @gregcaporaso suggesting a dynamic chunk size was caused by a bug. I fixed the bug, so we don't need a dynamic chunk size. But you could call it a dynamic chunk size if you want. The last chunk was always intended to contain the remainder of the samples, and if the chunk size is bigger than the number of samples then the first chunk is also the last chunk.